src/migrations/versions/0820dad794e6_delete_removed_status.py

```diff
@@ -0,0 +1,46 @@
+"""Delete removed status
+Revision ID: 0820dad794e6
+Revises: f552a27b4390
+Create Date: 2024-10-08 22:41:38.868185
+"""
+from alembic import op
+from alembic_postgresql_enum import TableReference
+
+# revision identifiers, used by Alembic.
+revision = "0820dad794e6"
+down_revision = "f552a27b4390"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute("UPDATE job SET status = 'cancelled' WHERE status = 'removed';")
```

Comment:
```suggestion
    op.execute("DELETE FROM job WHERE status = 'removed';")
```
actually this works too and is even better